reinforce/ppo/exploration.py

```
@@ -0,0 +1,257 @@
+        # ##>: Positive error = entropy too low, increase coef.
+        # Negative error = entropy too high, decrease coef.
+        adjustment = self.config.entropy_lr * entropy_error
+        self.entropy_coef = np.clip(
+            self.entropy_coef + adjustment,
+            self.config.min_entropy_coef,
+            self.config.max_entropy_coef,
+        )
+
+        return self.entropy_coef
+
+    def get_stage_entropy_coef(self, stage_index: int, num_stages: int = 4) -> float:
```

Ruff (0.14.10)

Unused method argument: `num_stages`

(ARG002)